cmd/cosign/cli/sign/sign.go

```diff
@@ -135,6 +134,13 @@ func SignCmd(ctx context.Context, ko KeyOpts, regOpts options.RegistryOptions, a
 		}
 	}
 
+	// TODO - Defaulting the timeout to zero as the CLI doesn't accept timeout.
```

Comment:
IDK how to interpret the work needed to address this TODO

Reply:
fixed it up